src/record/mutation.ts

```diff
@@ -359,13 +362,16 @@ export default class MutationBuffer {
       if (!node) {
         for (let index = addList.length - 1; index >= 0; index--) {
           const _node = addList.get(index)!;
```

Comment:
Since we are no longer expecting `_node` to always have a value, let's remove the `!` which marks it as non-null.
```suggestion
          const _node = addList.get(index);
```